plugins/formal-verification/formver.annotations/src/org/jetbrains/kotlin/formver/plugin/Annotations.kt

```diff
@@ -15,3 +15,7 @@ annotation class DumpExpEmbeddings
 annotation class Unique
 @Target(AnnotationTarget.VALUE_PARAMETER)
 annotation class Borrowed
+
+// We annotate the function to trigger the purity check, and if that succeeds, convert the function to a Viper function.
+annotation class Pure
+annotation class DebugPurity
```

Comment:
This would be better handled by a directive (see our tests directory for how those work)